test-result-summary-client/src/Build/PossibleIssues.jsx

```diff
@@ -15,6 +17,25 @@ export default class PossibleIssues extends Component {
         await this.fetchIssues();
     }
 
+    getUserFeedback = async(repoName, buildName, issueName, issueCreator, accuracy) => { 
+        const feedback = await fetchData(`/api/getFeedbackUrl?repoName=${repoName}&buildName=${buildName}&issueName=${issueName}&issueCreator=${issueCreator}&accuracy=${accuracy}`);
+        let feedbackRepo, feedbackBuild, feedbackIssue, feedbackCreator, feedbackAccuracy;
```

Comment:
Since API has been changed to return `error` or `success`, these variables are not needed.